blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/RollupType.java

```diff
@@ -47,6 +47,41 @@ else if (value instanceof BluefloodEnumRollup) {
             throw new Error(String.format("Cannot discern RollupType from %s", value.getClass().getSimpleName()));
     }
 
+    /**
+     *  Get RollupType from Rollup class type.
+     *
+     * @param rollupTypeClass
+     * @return RollupType
+     */
+    public static RollupType fromRollupTypeClass( Class<? extends Rollup> rollupTypeClass )  {
+
+        if (rollupTypeClass.equals(SimpleNumber.class)
+                || rollupTypeClass.equals( BasicRollup.class)
+                || rollupTypeClass.equals( String.class )) {
```

Comment:
@tilogaat @GeorgeJahad  - I added the check for String.class due to a error in a https://github.com/rackerlabs/blueflood/blob/enums-feature/blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupHandlerIntegrationTest.java#L125  

I'm not sure if this is the right thing to do.